package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "simple-local-avatars",
+    "version": "2.2.0",
+    "description": "Adds an avatar upload field to user profiles. Generates requested sizes on demand just like Gravatar!",
+    "scripts": {
```

Comment:
Hey @faisal-alvi I have added WP Scripts to build the package.json and you have something like this,
could you help me here what codes need to be kept.

```
"scripts": {
    "build": "uglifyjs simple-local-avatars.dev.js -o simple-local-avatars.js -c -m"
  },
```

Reply:
@ajmaurya99 we can use `prebuild` to uglify JS file and keep `build` for `wp-scripts build"`.

Reply:
We dont need `prebuild` command as we have a build command to minify the `js` files.